### DISCUSSION TITLE
STAC Version 2.1 Release

### DISCUSSION THREAD

**Discussion by Xylopyrographer** [Xylopyrographer/STAC#53] [Show and tell]
@mqshaw 

Good day Mark:

All's well in Boise-land with you and the family I pray.

Just pushed v2.1.

Pretty happy with the OTA implementation gotta say. And yes, not having to enter the IP address for setup and updating is so much better. Thanks for that great idea.

Display updates are improved with this release as I found a 🤦🏼‍♂️ in one of the drawing routines after the last push to the 2.1_dev branch.

Give it a go when you can. Feedback is always appreciated.

Blessings.

**Comment by mqshaw** [comment]
Hi Rob,

Thanks for the update.  Forgive me, its been a while since I left that comment.  Can you remind me what my idea was gain 😊   Was it to use a simple mDNS STAC URL when using the direct connect for setup?

Pretty happy with the OTA implementation gotta say. And yes, not having to enter the IP address for setup and updating is so much better. Thanks for that great idea.
I have been running the code for a few weeks now and it has been solid, but I am not sure if I am on the latest version.  I need to pull and re-deploy again to be sure.

Mark


From: Xylopyrographer ***@***.***>
Sent: Tuesday, February 14, 2023 11:57 AM
To: Xylopyrographer/STAC ***@***.***>
Cc: Mark Shaw ***@***.***>; Mention ***@***.***>
Subject: [Xylopyrographer/STAC] STAC Version 2.1 Release (Discussion #53)


@mqshaw<https://github.com/mqshaw>

Good day Mark:

All's well in Boise-land with you and the family I pray.

Just pushed v2.1.

Pretty happy with the OTA implementation gotta say. And yes, not having to enter the IP address for setup and updating is so much better. Thanks for that great idea.

Display updates are improved with this release as I found a 🤦🏼‍♂️ in one of the drawing routines after the last push to the 2.1_dev branch.

Give it a go when you can. Feedback is always appreciated.

Blessings.

**Comment by beamholder** [comment]
Hiya Mark. Yes, the idea from couple of years back to use mDNS. 😄

Cheers,
    Rob
================================



On Feb 20, 2023, at 5:55 PM, Mark Shaw ***@***.***> wrote:


Hi Rob, 

Thanks for the update. Forgive me, its been a while since I left that comment. Can you remind me what my idea was gain 😊 Was it to use a simple mDNS STAC URL when using the direct connect for setup? 

Pretty happy with the OTA implementation gotta say. And yes, not having to enter the IP address for setup and updating is so much better. Thanks for that great idea. 
I have been running the code for a few weeks now and it has been solid, but I am not sure if I am on the latest version. I need to pull and re-deploy again to be sure. 

Mark 


From: Xylopyrographer ***@***.***> 
Sent: Tuesday, February 14, 2023 11:57 AM 
To: Xylopyrographer/STAC ***@***.***> 
Cc: Mark Shaw ***@***.***>; Mention ***@***.***> 
Subject: [Xylopyrographer/STAC] STAC Version 2.1 Release (Discussion #53) 


@mqshaw<https://github.com/mqshaw> 

Good day Mark: 

All's well in Boise-land with you and the family I pray. 

Just pushed v2.1. 

Pretty happy with the OTA implementation gotta say. And yes, not having to enter the IP address for setup and updating is so much better. Thanks for that great idea. 

Display updates are improved with this release as I found a 🤦🏼‍♂️ in one of the drawing routines after the last push to the 2.1_dev branch. 

Give it a go when you can. Feedback is always appreciated. 

Blessings.

**Comment by mqshaw** [comment]
Hi Rob,

Just an FYI, I was able to get v2.1 loaded onto the two STAC’s at church yesterday, so I should be able to give you an update on their performance in a few weeks.  After I had completed the update I saw that you released 2.1.1, but it was rather minor changes so I chose not to update it to the latest at that time.

Mark

**Comment by beamholder** [comment]
Hi ya Mark:

Sounds good. The 2.1.1 release will let you test the OTA update feature 👍.

Cheers,
    Rob
================================



On Mar 13, 2023, at 4:17 PM, Mark Shaw ***@***.***> wrote:


Hi Rob, 

Just an FYI, I was able to get v2.1 loaded onto the two STAC’s at church yesterday, so I should be able to give you an update on their performance in a few weeks. After I had completed the update I saw that you released 2.1.1, but it was rather minor changes so I chose not to update it to the latest at that time. 

Mark
